doc/nrf/device_guides/working_with_nrf/nrf70/developing/power_profiling.rst

```diff
@@ -0,0 +1,81 @@
+.. _ug_nrf70_developing_power_profiling:
+
+Power profiling of nRF7002 DK
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Power Profiler Kit II (PPK2) of Nordic Semiconductor can be used in conjunction with the nRF7002 DK to measure the power consumption of nRF70 Series devices in Low-power mode.
```

Comment:
```suggestion
The Power Profiler Kit II (PPK2) of Nordic Semiconductor can be used in conjunction with the nRF7002 DK to evaluate the power consumption of nRF70 Series devices in low-power mode.
```

Reply:
Since Low-power mode is a specific mode, so as per https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120298016/Terms+abbreviations+and+spelling+rules, would like to keep it as is.